### DISCUSSION TITLE
Error removing .web/_static/server in deploy

### DISCUSSION THREAD

**Discussion by hpharmsen** [reflex-dev/reflex#3215] [Q&A] [answered]
I get errors while deploying. Somehow the reflex deploy statement tries to delete .web/_static/server but it is nog empty. See output below.

My command is
```bash
reflex deploy  --no-interactive -k electudetranslate -r fra
```
and then some environment variables.
The funny thing is: it always works when I immediately try the same command again.

```bash
Creating Production Build failed with exit code 1

> export-sitemap
> next build && next-sitemap

   Linting and checking validity of types ...
   ▲ Next.js 14.0.1

   Creating an optimized production build ...
 ✓ Compiled successfully
   Collecting page data ...
   Generating static pages (0/3) ...

 ✓ Generating static pages (3/3)
   Finalizing page optimization ...
   Collecting build traces ...

> Build error occurred
[Error: ENOTEMPTY: directory not empty, rmdir '/Users/hp/proj/reflextranslate/.web/_static/server'] {
  errno: -66,
  code: 'ENOTEMPTY',
  syscall: 'rmdir',
  path: '/Users/hp/proj/reflextranslate/.web/_static/server'
}
Run with --loglevel debug  for the full log.
Creating Production Build:  ━━━━━━━━━━━━━━━━━━━━━━╺━━━━━━━━━━━━━━━━━  56% 5/9 0:00:11
Unable to export due to: 1```

**Comment by picklelo** [comment]
Thanks for reporting, we will look into it. So it does work for you if you retry? Can you see if running just `reflex export` also causes this error?

**Comment by hpharmsen** [comment]
Yes it happens too when I just run `reflex export`.
And here too: just running  `reflex export` again helps.

I suspect it has something to do with interference with the app running on localhost. If I first stop the `reflex run` command, then all seems fine. Reversely, when I have the app running on localhost and I do a `reflex export` the local server gives the message `New packages detected: Updating app...` and my app stops running locally.

**Reply by picklelo** [reply, marked as answer]
@hpharmsen Ah got it that makes sense now. The website reloads whenever there's changes in the `.web` directory so doing an export while it's running may mess things up. There's probably a workaround for this, but I'd suggest just stopping the app before exporting or deploying as quick fix.
